public/video-ui/src/components/VideoPublishBar/VideoPublishBar.js

```diff
@@ -52,6 +53,9 @@ export default class VideoPublishBar extends React.Component {
     this.props.publishVideo();
   };
 
+  hasPublishedVideoUsages = () =>
+    this.props.usages.data.published.video.length > 0;
```

Comment:
I just started thinking about this and maybe we don't need this check at all now that we are going to allow embedding unpublished atoms and we are going to have warnings in composer when embedding unpublished atoms. Maybe we should trust our users to read these warnings. Maybe check this with editorial when we next catch up with them.

Reply:
we're adding a card for this and it will be in a separate PR